qa/acceptance/spec/centos/spec_helper.rb

```diff
@@ -0,0 +1,20 @@
+require 'runner-tool'
+require_relative '../../../rspec/helpers'
+require_relative '../../../rspec/matchers'
+require_relative '../config_helper'
+
+def vagrant_boxes
+  [
+    "centos-6",
+    "centos-7"
+  ]
+end
```

Comment:
This point here is only for filtering which boxes belongs to which execution, on next changes we could select here only 1 box.

not sure what we define in the vagrantfile will become later available here.